_scripts/deploy.sh

```diff
@@ -5,7 +5,7 @@
 
 cd "$(dirname "$0")" || exit 1
 
-export IMAGE_PREFIX=deis VERSION=2.0.0-alpha
+export IMAGE_PREFIX=deis VERSION=canary
```

Comment:
I like the idea, too. I don't have really strong opinions about the name, but I admit that I think canary is sorta cute. :bird: